### ISSUE TITLE
How to find my log file (tracker-server.log) on synology installation

### ISSUE THREAD

**Issue by GioBert** [traccar/traccar-docker#29]
Hello,
I installed traccar on my synology nas through docker (https://hub.docker.com/r/magnaz/traccar/)
The service is online and I find the server at the internal IP: http://192.168.1.103:55000/

But: How to find my device in log file (tracker-server.log)?
I'm not a great expert ... sorry !!

**Comment by ahmetbombaci** [comment]
@GioBert There are multiple ways for the UI.

I think default option is http://traccar.litvak.su/installation.html

If you have HomeAssistant, they have component for display too: https://www.home-assistant.io/components/traccar/

**Comment by corvankruchten** [comment]
Did you find it? I'm searching as well....

**Comment by dough29** [comment]
You can find log files in "/opt/traccar/logs/" folder within your container.

Try this command :

`docker exec -it <CONTAINER> tail -f /opt/traccar/logs/tracker-server.log`

**Comment by acasandreiflorin** [comment]
> Hello,
> I installed traccar on my synology nas through docker (https://hub.docker.com/r/magnaz/traccar/)
> The service is online and I find the server at the internal IP: http://192.168.1.103:55000/
> 
> But: How to find my device in log file (tracker-server.log)?
> I'm not a great expert ... sorry !!

Hi there,
After a bit of trial and error, I managed to have my Synology docker running traccar.
The way it worked for me was to create the folders **docker/traccar/logs** and **docker/traccar/conf** in the Synology interface with their corresponding mount path /opt/traccar/logs and /opt/traccar/conf as showed in the capture below:
![image](https://user-images.githubusercontent.com/57418446/86946732-1a873d00-c14b-11ea-9993-203d76c336ac.png)
Neverthless starting the container at this point it fails with an error that no traccar.xml file is found. To fix this I just saved the default ones traccar.xml and default.xml from the github in the conf folder docker/traccar/conf
Both files traccar.xml and default.xml can be found here: https://github.com/traccar/traccar/tree/master/setup 
Consider taking the content as raw in a notepad as save it in your synology.
Restarting the container will go fine after and you will find the logs under the docker/traccar/logs folder.
I hope this helps
Cheers,
Florin
